content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -203,7 +213,7 @@ Clusters without their own control plane are referred to as {{< gloss "remote cl
     caption="A service mesh with a primary and a remote cluster"
     >}}
 
-Instead of running control planes in primary clusters inside the mesh, a service mesh composed entirely of
+Instead of running shared control planes in primary clusters inside the mesh, a service mesh composed entirely of
```

Comment:
remove `shared`

Reply:
Agree with @nmittler. You could have a mesh with multiple primary clusters, so there's no sharing in that case.